### PR TITLE
update to v1.9.3

### PR DESCRIPTION
**Pull request by cbouss**
SciPy 1.9.3 is a bug-fix release with no new features compared to 1.9.1.
Release note: https://docs.scipy.org/doc/scipy/release.1.9.3.html
Repo: https://github.com/scipy/scipy/
Changes 1.9.1 1.9.3: https://github.com/scipy/scipy/compare/v1.9.1...v1.9.3

Configuration:
- https://github.com/scipy/scipy/blob/v1.9.3/pyproject.toml
- https://github.com/scipy/scipy/blob/v1.9.3/setup.py
- 
Changes:
- Update version to 1.9.3
- Update dependencies
- Remove skipped tests for issues that have been fixed.

We are still building with setuptools because the meson build is not stable yet.

### CODE REVIEW

**Review by varlackc — APPROVED** [review]
LGTM

**Review by AndrewVallette — APPROVED** [review]
LGTM!

**Review by ous8292 — APPROVED** [review]
LGTM

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "1.9.1" %}
+{% set version = "1.9.3" %}
 {% set numpy = "1.19" %}  # [py<=39]
 {% set numpy = "1.21" %}  # [py>39]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:3]: Should we use `numpy` **1.22** for `py==310` and `win`?  See https://github.com/scipy/scipy/blob/de80faf9d3480b9dbb9b888568b64499e0e70c19/pyproject.toml#L42

Comment (cbouss) [reply recipe/meta.yaml:3]: Their reason is specific to what the wheels on pypi contain. I don't see this affecting us.

**Pull request merged**